tests/ttnn/unit_tests/gtests/test_conv2d.cpp

```diff
@@ -155,8 +159,8 @@ TEST_P(Conv2DFixture, Conv2DCalculateCorrectly) {
         input_tensor = ttnn::permute(input_tensor, SmallVector<int64_t>{0, 2, 3, 1});
 
         // Run Conv2D
-        auto [output_tensor, output_height, output_width, weight_tensor_on_device, bias_tensor_on_device] =
-            conv2d::conv2d(
+        auto [output_tensor, output_dimensions] =
+            std::get<std::tuple<ttnn::Tensor, std::tuple<OutputHeight, OutputWidth>>>(ttnn::conv2d(
```

Comment:
std::get<std::tuple<ttnn::Tensor, std::tuple<OutputHeight, OutputWidth>>>(
Do we need this part?